src/signals/incident-management/containers/IncidentDetail/index.js

```diff
@@ -62,6 +64,24 @@ const Preview = styled.div`
   z-index: 1;
 `;
 
+export const formatWeekOrWorkdays = (days, isCalendarDays) => {
```

Comment:
Shouldn't it be better to put these helper functions in their own service/helper/something file/folder?

Reply:
It should be better if we would get this data from the backend so all this logic does not have to be implemented at all.

I agree and it crossed my mind while putting it here, I  have moved it to a new transform service for now.